README.md

```diff
@@ -1,2 +1,5 @@
-# .github
-Default community health files
+# Default community health files
+
+These files are used by default for subprojects that don't provide their own.
```

Comment:
Perhaps make this more Toit-centric. Change title to 'Default community health files for Toit' and 'used by default by projects in the [toitlang](https://github.com/toitlang) organization'.

Reply:
done.